api/proxy/docs/help_out.txt

```diff
@@ -16,6 +16,12 @@ GLOBAL OPTIONS:
    --help, -h     show help
    --version, -v  print the version
 
+   Arbitrum Custom DA JSON RPC Server
+
+   --arbitrum-da.addr value  Server listening address (default: "0.0.0.0") [$EIGENDA_PROXY_ARB_CUSTOM_DA_ADDR]
+   --arbitrum-da.enabled     Whether or not to enable Arbitrum Custom DA JSON RPC API (default: false) [$EIGENDA_PROXY_ARB_CUSTOM_DA_ENABLED]
+   --arbitrum-da.port value  Server listening port (default: 3100) [$EIGENDA_PROXY_ARB_CUSTOM_DA_PORT]
```

Comment:
are we sure that this json-rpc server is only arbitrum specific and that it couldnt ever be reused for other stacks?
Just wondering whether we should call these `--json-rpc.addr` instead of `--arbitrum-da.addr`